tests/databricks/test_notebook.py

```diff
@@ -197,7 +198,8 @@ def test_databricks_notebook_operator_without_taskgroup_existing_cluster(
             "libraries": [{"nb_index": {"package": "nb_package"}}],
             "timeout_seconds": 0,
             "email_notifications": {},
-        }
+        },
+        version="2.1",
```

Comment:
same here

Reply:
yes, just referenced the constant in a newer commit to address this comment.